django_mesh/models.py

```diff
@@ -191,4 +194,46 @@ class MockYoutube(models.Model):
     you = models.CharField(max_length=140, unique=True)
     yoh = models.CharField(max_length=140, unique=True)
     yot = models.TextField(default='')
-    yoj = models.TextField(default='')
\ No newline at end of file
+    yoj = models.TextField(default='')
+
+class Media(_Abstract):
+
+    MEDIA_TYPES = Choices(
+        (0, 'NONE',       'none',),
+        (1, 'IMAGE',      'image',),
+        (2, 'VIDEO',      'video',),
+    )
+
+    channel = models.ForeignKey(Channel)
+    objects = MediaQuerySet.as_manager()
+
+    upload_file = models.FileField(upload_to='uploads/%Y/%m/%d')
+
+    media_type = models.IntegerField(max_length=1, default=MEDIA_TYPES.NONE, choices=MEDIA_TYPES)
+
+    oembed_html = models.TextField(default='', blank=True)
+
+    media_height = models.SmallIntegerField(null=True)
+    media_width = models.SmallIntegerField(null=True)
+
+    thumbnail_height = models.SmallIntegerField(default=120)
+    thumbnail_width = models.SmallIntegerField(default=200)
+
+    @property    
+    def file_url(self):
+        return self.upload_file.url
+
+    def render(self):
+
+        embed = '<a href="%s"> %s </a>' % (self.file_url, self.file_url)
+
+        if self.media_type == Media.MEDIA_TYPES.IMAGE:
+            embed = '<img src="%s" alt="%s" style="width:%d%%; height:%d%%;">' % (self.file_url, self.title, self.media_width, self.media_height)
```

Comment:
Instead of using the "style" attribute, could you use "width" and "height" attributes? Normally we'd use CSS, but images are a special case and it helps speed up the webpage rendering.